mmv1/products/beyondcorp/SecurityGateway.yaml

```diff
@@ -135,3 +135,8 @@ properties:
     type: String
     description: Identifier. Name of the resource.
     output: true
+  - name: delegatingServiceAccount
+    type: String
+    description: |-
+      Output only. Service account used for operations that involve resources in consumer projects.
```

Comment:
no need to include `Output only.` in description since having the `output_only: true` as part of the field automatically adds it as part of the generation in the docs

Reply:
Done